### DISCUSSION TITLE
wslenv between linux and windows ?

### DISCUSSION THREAD

**Discussion by SheepDomination** [microsoft/WSL#4971] [General]
The WSLENV environment variable is it interchangeable where it is set; whether in the `bash.rc` or within the Windows environment; and it will be read between Windows and WSL ?

**Comment by DHowett-MSFT** [comment]
Yes.

**Comment by SheepDomination** [comment]
@DHowett-MSFT What takes precedence Windows Environment variables or WSLENV; and can you control if some variables inside WSLENV have priority over Windows Environment variables ? 

Also how do aliases work inside Windows from Bash ?
